apps/portal/examples/spring-boot-template/content/source/Dockerfile

```
@@ -0,0 +1,59 @@
+# Create a non-root user for security
+RUN addgroup -S appgroup && adduser -S appuser -G appgroup
+
+WORKDIR /app
+
+# Copy the fat JAR from the builder stage
+COPY --from=builder /workspace/build/libs/app.jar ./app.jar
+
+# Switch to non-root user
+USER appuser
+
+EXPOSE ${{ values.port }}
```

Hadolint (2.14.0)

[error] unexpected 'v'
expecting '\', A port optionally followed by the protocol, A range of ports optionally followed by the protocol, a new line followed by the next instruction, or at least one space

(DL1000)